Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -0,0 +1,465 @@
+
+<img src="../assets/edge-bridge-tutorial/assurance-edge-bridge-track-event.jpg" alt="Simulator tracking buttons" width="800"/>
+
+Notice the differences in event data structure and format between the two types of events: Analytics (left) vs Edge (right) via Edge Bridge extension
+The top level EventType is converted from a `generic.track` to `edge` (that is, Analytics generic track event -> Edge event) (`1`). The Edge Bridge extension also populates the standard XDM field for event type (`eventType`) in the event data payload. Also notice that the `contextdata` has moved from directly under `EventData` to under the generic Edge XDM `data` property (`2`).
+
+<img src="../assets/edge-bridge-tutorial/analytics-edge-bridge-conversion.jpg" alt="Comparison of event data between analytics and edge bridge events" width="1100"/>
+
+The two new top level properties `xdm` and `data` are standard Edge event properties that are part of the Edge platform's XDM schema-based system for event data organization that enables powerful, customizable data processing. However, because the `contextdata` is not yet mapped to an XDM schema, it is not in a usable form for the Edge platform. We will solve this issue by mapping the event data to an XDM schema in the next section.
+
+## Data prep mapping
+
+<details>
+  <summary> Data Prep background</summary><p>
+
+Data Prep is an Adobe Experience Platform service which maps and transforms data to the [Experience Data Model (XDM)](https://experienceleague.adobe.com/docs/experience-platform/xdm/home.html).  Data Prep is configured from a Platform enabled [datastream](https://experienceleague.adobe.com/docs/experience-platform/edge/datastreams/overview.html) to map source data from the Edge Bridge mobile extension to the Platform Edge Network.
+
+This guide covers how to map data sent from the Edge Bridge within the Data Collection UI.
+
+For a quick overview of the capabilities of Data Prep, watch the following [video](https://experienceleague.adobe.com/docs/platform-learn/data-collection/edge-network/data-prep.html).
+
+> **Note**
+> The following documentation provides a comprehensive overview of the Data Prep capabilities:
+> - [Data Prep overview](https://experienceleague.adobe.com/docs/experience-platform/data-prep/home.html)
+> - [Data Prep mapping functions](https://experienceleague.adobe.com/docs/experience-platform/data-prep/functions.html)
+> - [Handling data formats with Data Prep](https://experienceleague.adobe.com/docs/experience-platform/data-prep/data-handling.html)
+>
+
+</p></details>
+
+### Prerequisites
+Before creating a mapping, we need to create an XDM schema (what we are mapping to) and configure a datastream (where the data will go). In order to send data to the Edge Network, the datastream must be configured with the Adobe Experience Platform service.
```

Comment:
> In order to send data to the Edge Network, the datastream must be configured with the Adobe Experience Platform service.

For the tutorial I think we discussed sending the data to Analytics and not to Platform. So, in the datastream the Analytics service must be added and not the Platform service. If Platform is enabled, we'd also need to create a dataset, but to save time we didn't want to cover that step.